### PR TITLE
A

### LINKED ISSUE #17: Pull Branch A and Modify it and Push it

### PR DESCRIPTION
**Pull request by Joy-Prince**
own issue make changes in branch A

i have done my issue....

### CODE REVIEW

File: Apple Juice [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (melwyngabriel) [inline Apple Juice:1]: Add a paragraph stating what is apple juice

Comment (melwyngabriel) [inline Apple Juice:1]: Add a paragraph in this file

**Review by melwyngabriel — CHANGES_REQUESTED** [review]
Add a paragraph in the commented file

File: Apple Juice (hunk 1 above)
Comment (Joy-Prince) [reply Apple Juice:1]: i added a paragraph

**Review by melwyngabriel — APPROVED** [review]
Awesome

**Pull request merged**